### PR TITLE
JBPM-9175: Stunner - Usability improvements around selection / drag capabilities

### PR DESCRIPTION
**Pull request by handreyrc**
@romartin, @inodeman : Please, could you review this PR?

See https://issues.redhat.com/browse/JBPM-9175

Assembly with:
* https://github.com/kiegroup/kie-wb-common/pull/3319
* https://github.com/kiegroup/lienzo-core/pull/111
* https://github.com/kiegroup/lienzo-tests/pull/94

### CODE REVIEW

**Comment by handreyrc** [comment]
Jenkins execute full downstream build

**Comment by handreyrc** [comment]
Jenkins execute full downstream build

**Comment by hasys** [comment]
Jenkins execute full downstream build

**Comment by romartin** [comment]
hey @hasys @domhanak please need for feedback here. Thx!

File: src/test/java/com/ait/lienzo/client/core/shape/toolbox/ToolboxVisibilityExecutorsTest.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(LienzoMockitoTestRunner.class)
+public class ToolboxVisibilityExecutorsTest {
+
+    @Test
+    public void animationDelay() {
```
Comment (hasys) [inline src/test/java/com/ait/lienzo/client/core/shape/toolbox/ToolboxVisibilityExecutorsTest.java:13]: Test method name should have `test` prefix.

Comment (handreyrc) [reply src/test/java/com/ait/lienzo/client/core/shape/toolbox/ToolboxVisibilityExecutorsTest.java]: @hasys, Sure! It is already fixed.

**Comment by handreyrc** [comment]
Jenkins execute full downstream build

**Comment by handreyrc** [comment]
Jenkins execute full downstream build

**Comment by handreyrc** [comment]
Jenkins execute compile downstream build

**Pull request merged**